waiter/src/waiter/scheduler/kubernetes.clj

```diff
@@ -666,7 +672,7 @@
 (defn kubernetes-scheduler
   "Returns a new KubernetesScheduler with the provided configuration. Validates the
    configuration against kubernetes-scheduler-schema and throws if it's not valid."
-  [{:keys [authentication http-options max-patch-retries max-name-length orchestrator-name
+  [{:keys [authentication authorizer http-options max-patch-retries max-name-length orchestrator-name
```

Comment:
Should we add schema validation for `authorizer` in the pre-condition?

Reply:
Done. (`contains-kind-sub-map?`)